README.md

```diff
@@ -153,9 +188,11 @@ Write Ruby code that will create an instance of a person.
 
 Your answer...
 
+
 ```ruby
-Replace this with your answer
-```
+john = Person.new('John','73')
```

Comment:
Need to indicate which attributes are getting values of "John" and 73. So something like...

```rb
# option A
john = Person.new(name: "John", age: 73)
Person.save # .new does not automatically save, so gotta do this

# option B
Person.create(name: "John", age: 73) # .create = .new + .save
```